pom.xml

```
@@ -474,6 +474,8 @@
 					<connectionUrl>scm:git:https://github.com/pagopa/pagopa-ecommerce-commons.git</connectionUrl>
 					<scmVersionType>tag</scmVersionType>
 					<scmVersion>${pagopa-ecommerce-commons.version}</scmVersion>
+					<scmVersionType>branch</scmVersionType>
+					<scmVersion>CHK-1551-add-payment-token-validity-time</scmVersion>
```

```suggestion
					<scmVersionType>branch</scmVersionType>
					<scmVersion>CHK-1551-add-payment-token-validity-time</scmVersion>
```
To delete after commons merge